projects/migrations/0007_auto_20150830_2025.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Yes, I run makemigration a couple of times cause I wasn't sure if I needed at one point. Also I was wondering why most of the migration files have generic names (automatically created).
I can have another look and undo the migrations and run the last one and hopefully will make it cleaner.